cycledash/static/js/examine/components/VCFValidation.js

```diff
@@ -19,11 +19,14 @@ var VCFValidation = React.createClass({
         .map(v => <option value={v.id} key={v.uri}>{v.uri}</option>);
     return (
       <div id='vcf-validations'>
-        Compare to:
-        <select onChange={this.handleSelect}
-                value={this.props.compareToVcfId}>
-          {vcfs}
-        </select>
+        <dl className='dl-horizontal'>
```

Comment:
Do we need this wrapped in the `div`, or can we just add `vcf-validations` to the `dl`?